salesforce/samples/rest_api_usecases/search_record_by_string.bal

```diff
@@ -31,17 +31,20 @@ public function main(){
     };
 
     // Create Salesforce client.
-    sfdc:Client baseClient = checkpanic new(sfConfig);
-
-    int totalRecords = 0;
+    sfdc:Client baseClient = check new(sfConfig);
     string searchString = "FIND {WSO2 Inc}";
-    sfdc:SoslResult|sfdc:Error res = baseClient->searchSOSLString(searchString);
-
-    if res is sfdc:SoslResult {
-        log:printInfo(res.searchRecords.length().toString() + " Record Received");
-    }
-    else{
-        log:printError(res.message());
-    }
+    stream<record{}, error?> resultStream = check baseClient->searchSOSLString(searchString);
+    int count = check countStream(resultStream);
+    log:printInfo(string `${count} Record Received`);
+}
 
+isolated function countStream(stream<record{}, error?> resultStream) returns int|error{
+    int nLines = 0;
+    var _ = check from var i in resultStream
```

Comment:
Can't we replace `var _` with only  `_`  here?

Reply:
We can remove `var _` completely too. Fixed in 8c88af2857fa79a4d714f0fef58d93bab32cb3b9